psbt/src/construct/mod.rs

```diff
@@ -293,80 +294,88 @@ impl Psbt {
             }
         };
 
-        if change > 0 {
-            let change_derivation = [UnhardenedIndex::one(), change_index.into()];
-            let mut bip32_derivation = bmap! {};
-            let bip32_derivation_fn = |key: miniscript::ForEach<DerivationAccount>| {
-                let (pubkey, key_source) = key
-                    .as_key()
-                    .bip32_derivation(SECP256K1, &change_derivation)
-                    .expect("already tested descriptor derivation mismatch");
-                bip32_derivation.insert(pubkey, key_source);
-                true
-            };
+        if change > 0 || psbt_outputs.len() <= 0 {
+            println!(
+                "\n{}{}\n{}{}{}\n\n",
+                "WARNING: ".yellow(), 
+                " Consider uses change address output to avoid unspendable bitcoin.",
+                "Provably unspendable: ",
+                change.to_string().red(),
+                " stat(s)".red()
+            );
```

Comment:
The library code must not contain any prints. This checks should be moved to a command-line part.

Reply:
Sorry about that, I will fix this and push again.